movie_REST_API/movie_REST_API/App/ViewModel/ActorView.swift

```diff
@@ -5,29 +5,29 @@ import Foundation
 
 /// Получение данных об актерах
 final class ActorView {
```

Comment:
```suggestion
final class ActorView {
```
В этом классе вообще не понял че происходит от вью не наследуется а называется вью делает запросы тут Нетворк сервис надо все переделать